src/components/functionselect/functionselect.js

```diff
@@ -14,25 +14,50 @@ angular.module('vlui')
 
         scope.func = {
           selected: undefined,
-          list: [undefined]
+          list: {
+            aboveFold: [],
+            belowFold: [] // could be empty
+          }
         };
 
-        function getFns(type) {
+        // timeUnits for T
+        var timeUnits = {
+          aboveFold: ["year", "quarter", "month", "date", "day", "hours", "yearmonthdate"],
+          belowFold: ["hoursminutes", "hoursminutesseconds", "milliseconds", 
```

Comment:
Remove yeardate, yearday -- they are bad.  I'm removing them from Vega-Lite.  

yearmonthday => yearmonthdate  (same for longer version of this)

remove yearquartermonth, quarter month to reduce # of redundant options.